Pinta.Effects/Effects/CloudsEffect.cs

```diff
@@ -34,40 +34,44 @@ public sealed class CloudsEffect : BaseEffect
 
 	public CloudsData Data => (CloudsData) EffectData!;  // NRT - Set in constructor
 
-	public CloudsEffect ()
+	public IPaletteService palette { get; }
```

Comment:
Should this have been `private`?

Reply:
Yes, it should have been private. Thanks for catching it